test/fixtures/module-require-depth/one.js

```diff
@@ -1,11 +0,0 @@
-// Flags: --expose_internals
-'use strict';
-const assert = require('assert');
-const {
-  requireDepth
-} = require('internal/modules/cjs/helpers');
```

Comment:
☝️ Test above was likely borked because it restructured the `requireDepth` property.

Reply:
The test itself was working fine. The problem was the way `requireDepth` was received inside of `lib/internal/modules/cjs/loader.js`. It used destructuring, so the value was read once and never changed inside of the file where it had to be changed.